pkg/bridge/bridge.go

```diff
@@ -122,14 +169,26 @@ func (br *Bridge) ReturnAddr(returnee *Link, vidList []uint16) error {
 		return fmt.Errorf("could not set no master, error: %w, link: %s", err, returnee.Attrs().Name)
 	}
 
-	// transfer address
-	if err := transferAddr(&Link{Link: br.Bridge}, returnee); err != nil {
-		return fmt.Errorf("transfer address failed, error: %w, src link: %s, dst link: %s", err, br.Name, returnee.Attrs().Name)
+	mode, err := returnee.getIPv4Mode()
+	if err != nil {
+		return fmt.Errorf("get ipv4 mode of link %s failed, error: %w", returnee.Attrs().Name, err)
+	}
+
+	if mode == dhcp.Static {
+		return transferAddr(&Link{Link: br.Bridge}, returnee)
+	}
+
+	if mode == dhcp.DHCP {
+		// stop bridge dhcp client
+		br.DHCPSignal <- false
+		// start dhcp client for physical nic
+		if err := returnee.startDHCP(); err != nil {
+			return fmt.Errorf("start dhcp client for link %s failed, error: %w", br.Name, err)
+		}
 	}
```

Comment:
We are trying to return address to the NIC. Seems that the NIC is in `off` mode(not dhcp) at that moment.